index.html

```diff
@@ -28,253 +29,364 @@
               </a>
             </li>
             <li class="icon">
-              <a
-                href="https://www.linkedin.com/in/hafedefheij"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
+              <a href="#" target="_blank" rel="noopener noreferrer">
                 <i class="fa-brands fa-linkedin-in"></i>
               </a>
             </li>
             <li class="icon">
               <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-medium"></i>
+                <img src="./images/medium.png" alt="medium" />
               </a>
             </li>
             <li class="icon">
-              <a
-                href="https://github.com/hafedefheij"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
+              <a href="#" target="_blank" rel="noopener noreferrer">
                 <i class="fa-brands fa-github"></i>
               </a>
             </li>
             <li class="icon">
-              <a
-                href="https://angel.co/u/hafedefheij"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
+              <a href="#" target="_blank" rel="noopener noreferrer">
                 <i class="fa-brands fa-angellist"></i>
               </a>
             </li>
           </ul>
-        </li>
-      </ul>
+        </div>
+      </article>
     </section>
+    <!-- WORKS SECTION -->
     <section id="works-container">
       <ul id="works">
         <li class="work-card">
-          <img
-            src="images/work_1.jpg"
-            alt="work_tonic_daily_personalized_reads"
-          />
-          <h2>Tonic</h2>
-          <ul class="experience">
-            <li>Canopy</li>
-            <li>&#x2022;</li>
-            <li>Back End Dev</li>
-            <li>&#x2022;</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-          <div id="button-container">
-            <button type="button" class="btn">
-              <a href="#">See Project</a>
-            </button>
-          </div>
+          <article class="work-card__article">
+            <div class="work-card__header">
+              <picture>
+                <source
+                  srcset="images/work_1_lg.png"
+                  media="(min-width: 768px)"
+                />
+                <img
+                  src="images/work_1.jpg"
+                  alt="work_tonic_daily_personalized_reads"
+                />
+              </picture>
+            </div>
+            <div class="work-card__body">
+              <div class="elements">
+                <h1 class="work-card__title">Tonic</h1>
```

Comment:
- [OPTIONAL ] I would advise you to change this `h1` tag to any of the `hx (h2-h6)` tags. It is the convention and best practice to use only one `h1` tag on a webpage. It would be nice if you used any other `hx` tag. 👍